TipCatDotNet.Api/Infrastructure/QrCodeGenerator.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.IO;
+using System.Drawing;
+using System.Threading;
+using System.Threading.Tasks;
+using CSharpFunctionalExtensions;
+using QRCoder;
+using HappyTravel.AmazonS3Client.Services;
+
+namespace TipCatDotNet.Api.Infrastructure
+{
+    public class QrCodeGenerator : IQrCodeGenerator
+    {
+        public QrCodeGenerator(IAmazonS3ClientService client)
+        {
+            _client = client;
+        }
+
+
+        public async Task<Result<string>> Generate(string memberCode, CancellationToken cancellationToken)
+        {
+            var url = $"https://dev.tipcat.net/api/members/{memberCode}/pay";
```

Comment:
Following today's discussion here should be `https://dev.tipcat.net/{memberCode}/pay`